editoast/src/views/infra/routes.rs

```diff
@@ -124,7 +131,11 @@ struct RoutesFromNodesPositions {
     tag = "infra,routes",
     params(InfraIdParam, RouteTrackRangesParams),
     responses(
-        (status = 200, body = inline(Vec<RouteTrackRangesResult>), description = "Foreach route, all the track ranges in it or an error")
+        (
+            status = 200, 
+            body = inline(Vec<RouteTrackRangesResult>), 
+            description = "Foreach route, either the N tracks_ranges and the N - 1 switches inbetween the tracks_ranges, or an error"
+        )
```

Comment:
`/// ` documenting each variant of `RouteTrackRangesResult` would be more explicit imho